com.valvesoftware.Steam.appdata.xml

```diff
@@ -38,6 +38,9 @@
       <image type="source" width="1280" height="720">https://raw.githubusercontent.com/flathub/com.valvesoftware.Steam/master/com.valvesoftware.Steam.screenshot-2.png</image>
     </screenshot>
   </screenshots>
+  <metadata>
+    <value key="GnomeSoftware::popular-background">https://raw.githubusercontent.com/flathub/com.valvesoftware.Steam/master/com.valvesoftware.Steam-thumb.png</value>
```

Comment:
Where does this picture come from? Are we allowed to use it?

Reply:
Pixabay License
Free for commercial use
No attribution required
https://pixabay.com/photos/game-remote-gamer-video-2294201